kano/profiling_late.py

```diff
@@ -17,14 +17,20 @@
 from kano.logging import logger
 from kano.profiling import CONF_FILE
 
-# load the configuration file
-with open(CONF_FILE, 'r') as inp_conf:
-    conf = yaml.load(inp_conf)
+conf = None
 myProfile = cProfile.Profile()
 app_name = sys.argv[0]
 point_current = ""
 
 
+def load_config():
```

Comment:
Don't we need to tell something that it should be using this or is it never used anyway? I don't see the matching part.

Reply:
As luck would have it, this was being used in production. Fixed it here and 2 more PRs incoming 👍